diff -rup lohit-kannada-2.4.5/Lohit-Kannada.sfd lohit-kannada-2.4.5_mod/Lohit-Kannada.sfd
--- lohit-kannada-2.4.5/Lohit-Kannada.sfd	2010-03-23 13:11:18.000000000 +0530
+++ lohit-kannada-2.4.5_mod/Lohit-Kannada.sfd	2010-10-18 14:56:39.589732967 +0530
@@ -13,14 +13,13 @@ Descent: 404
 LayerCount: 2
 Layer: 0 1 "Back"  1
 Layer: 1 1 "Fore"  0
-NeedsXUIDChange: 1
 XUID: [1021 722 38139966 6691028]
 FSType: 0
 OS2Version: 1
 OS2_WeightWidthSlopeOnly: 0
 OS2_UseTypoMetrics: 1
 CreationTime: 1161073346
-ModificationTime: 1269256065
+ModificationTime: 1287393996
 PfmFamily: 33
 TTFWeight: 400
 TTFWidth: 5
@@ -71,7 +70,14 @@ Lookup: 1 0 0 "Single Substitution looku
 Lookup: 1 0 0 "Single Substitution lookup 17"  {"Single Substitution lookup 17"  } []
 Lookup: 260 0 0 "'blwm' Below Base Mark in Kannada lookup 0"  {"'blwm' Below Base Mark in Kannada lookup 0"  } ['blwm' ('knda' <'dflt' > ) ]
 Lookup: 258 0 0 "'kern' Horizontal Kerning in Kannada lookup 1"  {"'kern' Horizontal Kerning in Kannada lookup 1"  } ['kern' ('knda' <'dflt' > ) ]
+Lookup: 258 0 0 "'kern' Horizontal Kerning lookup 2"  {"'kern' Horizontal Kerning lookup 2-1"  } ['kern' ('DFLT' <'dflt' > 'deva' <'dflt' > 'knda' <'dflt' > 'latn' <'dflt' > ) ]
 DEI: 91125
+KernClass2: 4 2 "'kern' Horizontal Kerning lookup 2-1" 
+ 763 U0C96_U0CCB.blwf U0C99_U0CCB.blwf U0C9A_U0CCB.blwf U0C9B_U0CCB.blwf U0C9C_U0CCB.blwf U0CA1_U0CCB.blwf U0CA2_U0CCB.blwf U0CA3_U0CCB.blwf U0CA5_U0CCB.blwf U0CA6_U0CCB.blwf U0CA7_U0CCB.blwf U0CA8_U0CCB.blwf U0CAA_U0CCB.blwf U0CAB_U0CCB.blwf U0CAC_U0CCB.blwf U0CAD_U0CCB.blwf U0CAE_U0CCB.blwf U0CAF_U0CCB.blwf U0CB1_U0CCB.blwf U0CB2_U0CCB.blwf U0CB5_U0CCB.blwf U0C96_U0CCD.blwf U0C99_U0CCD.blwf U0C9A_0CCD.blwf U0C9B_U0CCD.blwf U0C9C_U0CCD.blwf U0C9E_U0CCD.blwf U0C9F_U0CCD.blwf U0CA1_U0CCD.blwf U0CA2_U0CCD.blwf U0CA3_U0CCD.blwf U0CA5_U0CCD.blwf U0CA6_U0CCD.blwf U0CA7_U0CCD.blwf U0CA8_U0CCD.blwf U0CAA_U0CCD.blwf U0CAB_U0CCD.blwf U0CAC_U0CCD.blwf U0CAD_U0CCD.blwf U0CAE_U0CCD.blwf U0CAF_U0CCD.blwf U0CB1_U0CCD.blwf U0CB2_U0CCD.blwf U0CB5_U0CCD.blwf U0CB8_U0CCD.blwf
+ 135 U0C98_U0CCB.blwf U0C9D_U0CCB.blwf U0CB7_U0CCB.blwf U0CB9_U0CCB.blwf U0C98_U0CCD.blwf U0C9D_U0CCD.blwf U0CB7_U0CCD.blwf U0CB9_U0CCD.blwf
+ 84 U0C97_U0CCB.blwf U0C95_U0CCD.blwf U0C97_U0CCD.blwf U0CB3_U0CCD.blwf U0CB6_U0CCD.blwf
+ 358 exclam quotedbl numbersign dollar percent ampersand quotesingle parenleft parenright asterisk plus comma hyphen period slash zero one two three four five six seven eight nine colon semicolon less equal greater question at bracketleft backslash bracketright asciicircum underscore grave braceleft bar braceright asciitilde cent multiply divide uni0964 uni0965
+ 0 {} 0 {} 0 {} 286 {} 0 {} 400 {} 0 {} 200 {}
 ChainSub2: coverage "'blws' Below Base Substitutions in Kannada lookup 9"  0 0 0 1
  1 1 0
   Coverage: 16 U0CB0_U0CCD.blwf
@@ -360,7 +366,7 @@ NameList: Adobe Glyph List
 DisplaySize: -36
 AntiAlias: 1
 FitToEm: 1
-WinInfo: 0 37 14
+WinInfo: 136 34 16
 Grid
 239 468 m 5
  239 452 243.5 438.333 252.5 427 c 132
@@ -47175,6 +47181,7 @@ SplineSet
  -191 -182 l 1,44,-1
  -191 -182 l 1,13,-1
 EndSplineSet
+LCarets2: 1 0 
 Ligature2: "'blwf' Below Base Forms in Kannada lookup 2" U0C95 U0CCB
 EndChar
 
@@ -47239,6 +47246,7 @@ SplineSet
  -234 -466 -234 -466 -234 -423 c 1,72,-1
  -234 -423 l 1,13,14
 EndSplineSet
+LCarets2: 1 0 
 Ligature2: "'blwf' Below Base Forms in Kannada lookup 2" U0C96 U0CCB
 EndChar
 
@@ -47267,6 +47275,7 @@ SplineSet
  -221 -475 l 1,18,-1
  -221 -475 l 1,0,-1
 EndSplineSet
+LCarets2: 1 0 
 Ligature2: "'blwf' Below Base Forms in Kannada lookup 2" U0C97 U0CCB
 EndChar
 
@@ -47331,6 +47340,7 @@ SplineSet
  -234 -379 -234 -379 -234 -325 c 1,68,-1
  -234 -325 l 1,7,8
 EndSplineSet
+LCarets2: 1 0 
 Ligature2: "'blwf' Below Base Forms in Kannada lookup 2" U0C98 U0CCB
 EndChar
 
@@ -47407,6 +47417,7 @@ SplineSet
  -221 -458 -221 -458 -221 -410 c 1,87,-1
  -221 -410 l 1,5,6
 EndSplineSet
+LCarets2: 1 0 
 Ligature2: "'blwf' Below Base Forms in Kannada lookup 2" U0C99 U0CCB
 EndChar
 
@@ -47478,6 +47489,7 @@ SplineSet
  -227 -458 -227 -458 -227 -410 c 1,81,-1
  -227 -410 l 1,0,1
 EndSplineSet
+LCarets2: 1 0 
 Ligature2: "'blwf' Below Base Forms in Kannada lookup 2" U0C9A U0CCB
 EndChar
 
@@ -47674,6 +47686,7 @@ SplineSet
  -234 -319 -234 -319 -234 -254 c 1,68,-1
  -234 -254 l 1,13,14
 EndSplineSet
+LCarets2: 1 0 
 Ligature2: "'blwf' Below Base Forms in Kannada lookup 2" U0C9D U0CCB
 EndChar
 
@@ -47918,6 +47931,7 @@ SplineSet
  137 -288 137 -288 137 -293 c 1,70,-1
  137 -293 l 1,58,59
 EndSplineSet
+LCarets2: 1 0 
 Ligature2: "'blwf' Below Base Forms in Kannada lookup 2" U0CA1 U0CCB
 EndChar
 
@@ -47990,6 +48004,7 @@ SplineSet
  137 -288 137 -288 137 -293 c 1,77,-1
  137 -293 l 1,65,66
 EndSplineSet
+LCarets2: 1 0 
 Ligature2: "'blwf' Below Base Forms in Kannada lookup 2" U0CA2 U0CCB
 EndChar
 
@@ -48056,6 +48071,7 @@ SplineSet
  -234 -427 -234 -427 -234 -384 c 1,76,-1
  -234 -384 l 1,0,1
 EndSplineSet
+LCarets2: 1 0 
 Ligature2: "'blwf' Below Base Forms in Kannada lookup 2" U0CA3 U0CCB
 EndChar
 
@@ -48091,6 +48107,7 @@ SplineSet
  -182 -207 -182 -207 -182 -234 c 1,28,-1
  -182 -234 l 1,16,17
 EndSplineSet
+LCarets2: 1 0 
 Ligature2: "'blwf' Below Base Forms in Kannada lookup 2" U0CA4 U0CCB
 EndChar
 
@@ -48146,6 +48163,7 @@ SplineSet
  -156 -260 -156 -260 -156 -325 c 1,50,-1
  -156 -325 l 1,31,32
 EndSplineSet
+LCarets2: 1 0 
 Ligature2: "'blwf' Below Base Forms in Kannada lookup 2" U0CA5 U0CCB
 EndChar
 
@@ -48187,6 +48205,7 @@ SplineSet
  -156 -260 -156 -260 -156 -325 c 1,38,-1
  -156 -325 l 1,19,20
 EndSplineSet
+LCarets2: 1 0 
 Ligature2: "'blwf' Below Base Forms in Kannada lookup 2" U0CA6 U0CCB
 EndChar
 
@@ -48236,6 +48255,7 @@ SplineSet
  -156 -260 -156 -260 -156 -325 c 1,45,-1
  -156 -325 l 1,26,27
 EndSplineSet
+LCarets2: 1 0 
 Ligature2: "'blwf' Below Base Forms in Kannada lookup 2" U0CA7 U0CCB
 EndChar
 
@@ -48288,6 +48308,7 @@ SplineSet
  -234 -334 -234 -334 -234 -286 c 1,54,-1
  -234 -286 l 1,13,14
 EndSplineSet
+LCarets2: 1 0 
 Ligature2: "'blwf' Below Base Forms in Kannada lookup 2" U0CA8 U0CCB
 EndChar
 
@@ -48814,6 +48835,7 @@ SplineSet
  -234 -256 -234 -256 -234 -182 c 1,69,-1
  -234 -182 l 1,26,27
 EndSplineSet
+LCarets2: 1 0 
 Ligature2: "'blwf' Below Base Forms in Kannada lookup 2" U0CB3 U0CCB
 EndChar
 
@@ -48921,6 +48943,7 @@ SplineSet
  -234 -414 -234 -414 -234 -371 c 1,59,-1
  -234 -371 l 1,16,17
 EndSplineSet
+LCarets2: 1 0 
 Ligature2: "'blwf' Below Base Forms in Kannada lookup 2" U0CB6 U0CCB
 EndChar
 
@@ -49058,6 +49081,7 @@ SplineSet
  104 -224 104 -224 104 -267 c 1,50,-1
  104 -267 l 1,38,39
 EndSplineSet
+LCarets2: 1 0 
 Ligature2: "'blwf' Below Base Forms in Kannada lookup 2" U0CB9 U0CCB
 EndChar
 
@@ -51329,6 +51353,7 @@ SplineSet
  -221 -475 l 1,0,-1
 EndSplineSet
 Kerns2: 24 -219 "'kern' Horizontal Kerning in Kannada lookup 1" 
+LCarets2: 1 0 
 Ligature2: "'blwf' Below Base Forms in Kannada lookup 2" U0C97 U0CCD
 EndChar
 
@@ -51545,6 +51570,7 @@ SplineSet
  -227 -458 -227 -458 -227 -410 c 1,77,-1
  -227 -410 l 1,0,1
 EndSplineSet
+LCarets2: 1 0 
 Ligature2: "'blwf' Below Base Forms in Kannada lookup 2" U0C9A U0CCD
 EndChar
 
@@ -51998,6 +52024,7 @@ SplineSet
  137 -288 137 -288 137 -293 c 1,69,-1
  137 -293 l 1,57,58
 EndSplineSet
+LCarets2: 1 0 
 Ligature2: "'blwf' Below Base Forms in Kannada lookup 2" U0CA1 U0CCD
 EndChar
 
@@ -52176,6 +52203,7 @@ SplineSet
  -182 -207 -182 -207 -182 -234 c 1,23,-1
  -182 -234 l 1,14,15
 EndSplineSet
+LCarets2: 1 0 
 Ligature2: "'blwf' Below Base Forms in Kannada lookup 2" U0CA4 U0CCD
 EndChar
 
@@ -52482,6 +52510,7 @@ SplineSet
  -234 -379 -234 -379 -234 -325 c 1,50,-1
  -234 -325 l 1,7,8
 EndSplineSet
+LCarets2: 1 0 
 Ligature2: "'blwf' Below Base Forms in Kannada lookup 2" U0CAB U0CCD
 EndChar
 
@@ -52657,6 +52686,7 @@ SplineSet
  -234 -366 -234 -366 -234 -280 c 1,49,-1
  -234 -280 l 1,14,15
 EndSplineSet
+LCarets2: 1 0 
 Ligature2: "'blwf' Below Base Forms in Kannada lookup 2" U0CAE U0CCD
 EndChar
 
@@ -52740,6 +52770,7 @@ SplineSet
  -778 -179 -778 -179 -778 35 c 2,18,-1
  -778 39 l 1,0,-1
 EndSplineSet
+LCarets2: 1 0 
 Substitution2: "Single Substitution lookup 17" glyph598
 Substitution2: "Single Substitution lookup 16" glyph596
 Substitution2: "Single Substitution lookup 15" glyph597
@@ -53126,6 +53157,7 @@ SplineSet
  -234 -475 l 1,32,-1
  -234 -475 l 1,0,1
 EndSplineSet
+LCarets2: 1 0 
 Ligature2: "'blwf' Below Base Forms in Kannada lookup 2" U0CB8 U0CCD
 EndChar
 
