diff -rup lohit-kannada-2.4.5/Lohit-Kannada.sfd lohit-kannada-2.4.5_mod/Lohit-Kannada.sfd
--- lohit-kannada-2.4.5/Lohit-Kannada.sfd	2011-07-05 11:27:01.588078733 +0530
+++ lohit-kannada-2.4.5_mod/Lohit-Kannada.sfd	2011-07-05 11:25:26.542160730 +0530
@@ -13,13 +13,14 @@ Descent: 404
 LayerCount: 2
 Layer: 0 1 "Back"  1
 Layer: 1 1 "Fore"  0
+NeedsXUIDChange: 1
 XUID: [1021 722 38139966 6691028]
 FSType: 0
 OS2Version: 1
 OS2_WeightWidthSlopeOnly: 0
 OS2_UseTypoMetrics: 1
 CreationTime: 1161073346
-ModificationTime: 1290486476
+ModificationTime: 1309845324
 PfmFamily: 33
 TTFWeight: 400
 TTFWidth: 5
@@ -369,13 +370,12 @@ EndShort
 LangName: 1033 "" "" "Regular" "FontForge 1.0 : Lohit Kannada : 19-7-2006" "" "Version 2.4.5" 
 GaspTable: 1 65535 2
 Encoding: UnicodeBmp
-Compacted: 1
 UnicodeInterp: none
 NameList: Adobe Glyph List
 DisplaySize: -36
 AntiAlias: 1
 FitToEm: 1
-WinInfo: 0 34 16
+WinInfo: 8330 34 14
 Grid
 239 468 m 5
  239 452 243.5 438.333 252.5 427 c 132
@@ -442,7 +442,7 @@ Grid
 EndSplineSet
 TeXData: 1 0 0 323584 161792 107861 0 1048576 107861 783286 444596 497025 792723 393216 433062 380633 303038 157286 324010 404750 52429 2506097 1059062 262144
 AnchorClass2: "Anchor-0"  "'blwm' Below Base Mark in Kannada lookup 0" 
-BeginChars: 66047 651
+BeginChars: 66047 652
 
 StartChar: .notdef
 Encoding: 65536 -1 0
@@ -55061,5 +55061,37 @@ SplineSet
  499 107 l 1,18,19
 EndSplineSet
 EndChar
+
+StartChar: uni20B9
+Encoding: 8377 8377 651
+Width: 631
+Flags: HW
+LayerCount: 2
+Fore
+SplineSet
+358 500 m 0,0,1
+ 359 509 359 509 359 514 c 1,2,-1
+ 59 514 l 1,3,-1
+ 85 577 l 1,4,-1
+ 346 577 l 1,5,6
+ 340 589 340 589 331 599 c 0,7,8
+ 280 654 280 654 165 654 c 2,9,-1
+ 59 654 l 1,10,-1
+ 85 717 l 1,11,-1
+ 583 717 l 1,12,-1
+ 557 654 l 1,13,-1
+ 367 654 l 1,14,15
+ 406 631 406 631 420 577 c 1,16,-1
+ 583 577 l 1,17,-1
+ 557 514 l 1,18,-1
+ 429 514 l 1,19,20
+ 429 358.613 429 358.613 171 340 c 1,21,-1
+ 507 0 l 1,22,-1
+ 407 0 l 1,23,-1
+ 70 337 l 1,24,-1
+ 70 396 l 1,25,26
+ 345.891 394 345.891 394 358 500 c 0,0,1
+EndSplineSet
+EndChar
 EndChars
 EndSplineFont
