diff -rup lohit-kannada-2.4.5/Lohit-Kannada.sfd lohit-kannada-2.4.5_mod/Lohit-Kannada.sfd
--- lohit-kannada-2.4.5/Lohit-Kannada.sfd	2012-01-23 19:14:13.273973865 +0530
+++ lohit-kannada-2.4.5_mod/Lohit-Kannada.sfd	2012-01-23 18:48:03.731100173 +0530
@@ -20,7 +20,7 @@ OS2Version: 1
 OS2_WeightWidthSlopeOnly: 0
 OS2_UseTypoMetrics: 1
 CreationTime: 1161073346
-ModificationTime: 1309845324
+ModificationTime: 1327324664
 PfmFamily: 33
 TTFWeight: 400
 TTFWidth: 5
@@ -375,7 +375,7 @@ NameList: Adobe Glyph List
 DisplaySize: -36
 AntiAlias: 1
 FitToEm: 1
-WinInfo: 8330 34 14
+WinInfo: 0 34 14
 Grid
 239 468 m 5
  239 452 243.5 438.333 252.5 427 c 132
@@ -1690,7 +1690,7 @@ EndChar
 
 StartChar: U0C97
 Encoding: 3223 3223 25
-Width: 620
+Width: 650
 GlyphClass: 2
 Flags: W
 AnchorPoint: "Anchor-0" 388 -21 basechar 0
@@ -19918,9 +19918,10 @@ EndChar
 
 StartChar: U0CB8_U0CC6.abvs
 Encoding: 65706 -1 266
-Width: 545
+Width: 572
 GlyphClass: 2
 Flags: W
+AnchorPoint: "Anchor-0" 265 -31 basechar 0
 LayerCount: 2
 Fore
 SplineSet
@@ -19978,6 +19979,7 @@ SplineSet
  51 585 l 1,68,-1
  51 585 l 1,38,-1
 EndSplineSet
+LCarets2: 1 0 
 Ligature2: "'abvs' Above Base Substitutions in Kannada lookup 0" U0CB8 U0CC6
 EndChar
 
@@ -55065,7 +55067,7 @@ EndChar
 StartChar: uni20B9
 Encoding: 8377 8377 651
 Width: 631
-Flags: HW
+Flags: W
 LayerCount: 2
 Fore
 SplineSet
